resources/enclosure.rb

```diff
@@ -16,6 +16,9 @@
 property :refresh_state, String, default: 'RefreshPending'
 property :options, Hash, default: {}
 
+# API300 or greater
```

Comment:
I'd drop the comment and leave the property together with the other ones, or use an in-line comment.